python/pythonmonkey/builtin_modules/XMLHttpRequest.js

```diff
@@ -7,11 +7,41 @@
  * 
  * @copyright Copyright (c) 2023 Distributive Corp.
  */
+'use strict';
 
 const { EventTarget, Event } = require('event-target');
 const { DOMException } = require('dom-exception');
 const { URL, URLSearchParams } = require('url');
 const { request, decodeStr } = require('XMLHttpRequest-internal');
+const debug = globalThis.python.eval('__import__("pythonmonkey").bootstrap.require')('debug');
+
+debug('xhr:module')('loaded XMLHttpRequest.js module');
+
+/**
+ * Truncate a string-like thing for display purposes, returning a string.
+ * @param {any}     what     The thing to truncate; must have a slice method and index property.
+ *                           Works with string, array, typedarray, etc.
+ * @param {number}  maxlen   The maximum length for truncation
+ * @param {boolean} coerce   Not false = coerce to printable character codes  
+ * @returns {string}
+ */
+function trunc(what, maxlen, coerce)
+{
+  if (coerce !== false && typeof what !== 'string')
+  {
+    what = Array.from(what).map(x => {
+      if (x > 31 && x < 127)
+        return String.fromCharCode(x);
+      else if (x < 32)
+        return eval('"\\u24' + ((x).toString(16)).padStart(2,0) + '"')
```

Comment:
We should avoid using `eval`.

Reply:
The [Control Pictures unicode block](https://en.wikipedia.org/wiki/Control_Pictures) starts at code point `0x2400`.
How about changing this to
```suggestion
        return String.fromCharCode(0x2400 + Number(x));
```

Reply:
Hey! Great suggestion. Much better.